index.html

```diff
@@ -5,6 +5,7 @@
     <meta name="viewport" content="width=device-width, initial-scale=1">
```

Comment:
**TODO FOR LOST POINTS**

Have some placeholder images somewhere so you can get an idea how real images will fit in your portfolio. Here is a tool I like to use as a suggestion for placeholder images.

https://placehold.it/